### DISCUSSION TITLE
Craft 4 option to disable default element sources?

### DISCUSSION THREAD

**Discussion by gregorterrill** [craftcms/cms#10676] [Ideas] [answered]
There are some situations I can envision where the new Custom Source options make the default section sources redundant, and it would be nice to be able to remove/hide them. As a simple example, let's say I have a **Partners** section with two entry types: **Company** and **Individual**. By default, there will be an element source for Partners that shows both entry types. I might prefer to have Partners shown as a heading, with two custom sources beneath it, one showing only the Company entry type and one showing only the Individual entry type. It's awesome that I can do that, but now I still have this default Partners source showing that's now redundant. I think having a  _Hidden_ or _Disabled_ lightswitch option for element sources would be the preferable solution, so that the source doesn't show up when looking at the Entries screen normally, but when I go to customize my element sources I still see all the default sections (the hidden ones are just greyed out or marked _Hidden_ or _Disabled_). I think it might cause some confusion if you were able to fully delete the defaults, because then it might be unclear what sections actually exist.

**Comment by brandonkelly** [comment, marked as answer]
Just got around to making this possible, for the next Craft 4 Beta release! Now native sources have “Enabled” settings, which can be turned off to prevent the source from showing up elsewhere in the control panel. (3cad4da23fd71e1b14a9693c821e5acc2a8d17f8)

<img width="472" alt="The new “Enabled” setting on a native element source" src="https://user-images.githubusercontent.com/47792/161686266-e2ff7ee3-dfc9-4310-a178-3721a6ec1814.png">

**Reply by brandonkelly** [reply]
4.0.0-beta.4 is out with this change!
